### PR TITLE
Updated relationship_type case manager - case manager to case manager- case , Migrated all the misaligned cases in relationships

### PR DESCRIPTION
**Pull request by patryllus**
Screenshot from 2024-08-01 14-37-49

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/kenyaemr/metadata/CommonMetadata.java [hunk 1]
```diff
@@ -435,7 +435,7 @@ public void install() {
 		install(relationshipType("Partner", "Partner", "Someone I had sex with for fun without commitment to a relationship", _RelationshipType.PARTNER));
 		install(relationshipType("Co-wife", "Co-wife", "Female member spouse in a polygamist household", _RelationshipType.CO_WIFE));
 		install(relationshipType("SNS", "SNS", "Social Network Strategy", _RelationshipType.SNS));
-		install(relationshipType("Case manager", "Case manager", "Case manager", _RelationshipType.CASE_MANAGER));
+		install(relationshipType("Case manager", "Case", "Case manager", _RelationshipType.CASE_MANAGER));
```
Comment (ojwanganto) [inline api/src/main/java/org/openmrs/module/kenyaemr/metadata/CommonMetadata.java:438]: Let's go with Case manager and client. That is the guidance I got

File: api/src/main/resources/AlignCaseManagerCaseRelationships.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
+SET FOREIGN_KEY_CHECKS=0;
+
+UPDATE relationship_type SET b_is_to_a = 'Case' WHERE uuid = '9065e3c6-b2f5-4f99-9cbf-f67fd9f82ec5';
```
Comment (ojwanganto) [inline api/src/main/resources/AlignCaseManagerCaseRelationships.sql:4]: Change this to client

File: api/src/main/resources/AlignCaseManagerCaseRelationships.sql [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+UPDATE relationship r
```
Comment (ojwanganto) [inline api/src/main/resources/AlignCaseManagerCaseRelationships.sql:6]: Do we want to limit this to the entries created using 3.x? I know we already had this relationship before but am not sure how this was implemented in 2.x

Comment (patryllus) [reply api/src/main/resources/AlignCaseManagerCaseRelationships.sql:6]: The second query below that one does the 2x migrations - looks like this: 
UPDATE relationship r
	INNER JOIN relationship_type t ON r.relationship = t.relationship_type_id
	SET r.person_a = r.person_b , r.person_b = r.person_a
    WHERE  t.uuid = '9065e3c6-b2f5-4f99-9cbf-f67fd9f82ec5';

**Review by ojwanganto — APPROVED** [review]

**Pull request merged**